### PR TITLE
Added more ship names for Amazon tribes

### PR DESCRIPTION
**Pull request by alasiax**
Based on rivers in the Amazon Basin for naming consistency.

### CODE REVIEW

File: data/tribes/ships/amazons/init.lua [hunk 1]
```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
...
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Amanã"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Amapá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Amazon"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Andirá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Andirá"),
```
Comment (frankystone) [inline data/tribes/ships/amazons/init.lua]: Two of same name

File: data/tribes/ships/amazons/init.lua [hunk 2]
```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
...
+      pgettext("shipname", "Arara"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Ararirá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arauá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arauá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arauã"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
```
Comment (frankystone) [inline data/tribes/ships/amazons/init.lua]: To of same name, last differs only with the hyphen on last 'a'. I think during a game no one can distinguish between apostroph and hyphen?

File: data/tribes/ships/amazons/init.lua [hunk 3]
```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
...
+      pgettext("shipname", "Coti"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Cubate"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Cuiari"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Cuieiras"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Cuieiras"),
```
Comment (frankystone) [inline data/tribes/ships/amazons/init.lua]: Two of same name.

File: data/tribes/ships/amazons/init.lua [hunk 4]
```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
...
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Içana"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Inauini"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Ipixuna"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Ipixuna"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Ipixuna"),
```
Comment (frankystone) [inline data/tribes/ships/amazons/init.lua]: Three of same name.

File: data/tribes/ships/amazons/init.lua [hunk 5]
```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
...
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Parauari"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Pardo"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Pati"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Pauini"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Pauini"),
```
Comment (frankystone) [inline data/tribes/ships/amazons/init.lua]: Two of same name.

File: data/tribes/ships/amazons/init.lua [hunk 6]
```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
...
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Piorini"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Piratucu"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Pitinga"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Preto"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Preto"),
```
Comment (frankystone) [inline data/tribes/ships/amazons/init.lua]: Two of same name.

**Review by frankystone — CHANGES_REQUESTED** [review]
Thanks for contributing!

There are some double entries, please fix.

**Comment by alasiax** [comment]
Thanks Frankystone :o) Updates made as requested.

**Review by frankystone — APPROVED** [review]
LGTM :-)

Let's wait for the checks to finish.

**Comment by hessenfarmer** [comment]
Nice work thanks a lot

**Pull request merged**